### DISCUSSION TITLE
Use a nvim colorscheme plugin.

### DISCUSSION THREAD

**Discussion by nitanmarcel** [NvChad/NvChad#1352] [Q&A] [answered]
I'm trying to use an installed colorscheme plugin for neovim and I can't find how to automatically apply it.

Doing `:colorscheme monochrome` sets it but only for the session. I've tried using `vim.cmd("colorscheme monochrome")` in the plugin config, `custom/init.lua`, `custom/chadrc.lua` but it didn't worked.

**Comment by siduck** [comment, marked as answer]
@nitanmarcel you cannot use other colorschemes in nvchad as it might break many plugin configs, each plugin config loads its highlight groups from base46 repo so either you have to replace default configs of all plugins in nvchad or submit request for the theme you want in base46 repo
